Event-Triggered Journeys Steps [DOC-1024] (#7396)
segmentio/segment-docs: src/engage/journeys/event-triggered-journeys-steps.md

Here’s how to configure this step within a journey:

1. Select and name the step:
   - Choose the destination for the data.
   - (Optional:) Assign a unique name for clarity on the journey canvas.
2. Choose the action:
   - Define the change to trigger in the destination, like updating a record.
   - For Destination Functions, the behavior is defined in the function code, so no action selection is needed.
3. Configure and map the event:
   - Name the event sent to the destination.
   - Add profile traits to include in the payload.
   - View a payload preview to map [journey context attributes](/docs/engage/journeys/journey-context/#send-to-destination) to destination fields.
   - Test the payload to ensure proper delivery and validation.

Before activating the journey, **send a test event to verify that the payload matches your expectations** and that it reaches the destination successfully.

### Destination event payload schema

The events that Segment sends to destinations from Event-Triggered Journeys include an object called `journey_context` within the event’s properties. The `journey_context` object contains:

> - The triggering event that started the journey, unless it was replaced by a new event in a Hold Until step.

Reviewer: ```suggestion
- The event that triggered the journey, unless it was replaced by a new event in a Hold Until step.
```

  "originalTimestamp": "2025-01-15T02:02:15.908Z",
  "context": {
    "personas": {
      "computation_class": "journey_step",
      "computation_id": "journey_name__step_name_8943l",
      "computation_key": "journey_name__step_name_8943l",
      "event_emitter_id": "event_tester_lekqCASsZX",
      "namespace": "spa_w5akhv1XwnGj5j2HVT6NWX",
      "space_id": "spa_w5akhv1XwnGj5j2HVT6NWX"
    }
  },
  "properties": {
    "journey_context": {
      "triggering_event": {
        "organization": "Pied Piper",
        "compression_ratio": 5.2,
        "output_code": "not_hotdog"
      },
      "event_from_hold_until_step": {
        "organization": "Tres Commas",
        "user_name": "Russ Hanneman",
        "output_code": "always_blue"
      }
    },
    "journey_metadata": {
      "journey_id": "2GKsjADZkD",
      "epoch_id": "yiC2qPZNIS"
    },
    "user_name": "Richard Hendricks",
    "coding_style": "tabs_only",
    "pivot_count": 12
  },
  "messageId": "personas_up0crko4htawmo2c9ziyq"
}
```

This example shows how data is structured and enriched with contextual details so that destinations receive the information they need to act effectively.

### Managing activations

> Activations control the configuration for sending data to destinations, including the destination type, selected action, and mapped attributes. Managing activations allows you to adjust how data flows to a destination without altering the overall journey logic.

Reviewer: ```suggestion
Activations control the configuration for sending data to destinations, including the destination type, selected action, and mapped attributes. Managing activations allow you to adjust how data flows to a destination without altering the overall journey logic.
```

#### Editing activations

> You can make updates to an existing activation to align mapped attributes with changes in the downstream schema, as well as to add or remove profile traits included in the payload.

Reviewer: ```suggestion
You can make updates to an existing activation to align mapped attributes with changes in the downstream schema and add or remove profile traits included in the payload.
```
